pkg/config/consistent.go

```diff
@@ -13,10 +13,44 @@
 
 package config
 
-// ConsistentConfig represents replication consistency config for a changefeed
+import (
+	"fmt"
+
+	"github.com/pingcap/tidb/br/pkg/storage"
+	cerror "github.com/pingcap/tiflow/pkg/errors"
+	"github.com/pingcap/tiflow/pkg/redo"
+)
+
+const (
+	// MinFlushIntervalInMs is the minimum value of flush interval, which is set
+	// to two seconds to reduce the frequency of accessing external storage.
+	MinFlushIntervalInMs = 2000
+)
+
+// ConsistentConfig represents replication consistency config for a changefeed.
 type ConsistentConfig struct {
 	Level             string `toml:"level" json:"level"`
 	MaxLogSize        int64  `toml:"max-log-size" json:"max-log-size"`
 	FlushIntervalInMs int64  `toml:"flush-interval" json:"flush-interval"`
 	Storage           string `toml:"storage" json:"storage"`
 }
+
+// ValidateAndAdjust validates the consistency config and adjusts it if necessary.
+func (c *ConsistentConfig) ValidateAndAdjust() error {
+	if !redo.IsConsistentEnabled(c.Level) {
+		return nil
+	}
+
+	if c.FlushIntervalInMs < MinFlushIntervalInMs {
+		return cerror.ErrInvalidReplicaConfig.FastGenByArgs(
+			fmt.Sprintf("The consistent.flush-interval:%d must be equal or greater than %d",
```

Comment:
It might be a breaking change, also the official document recommends 500-2000.

https://github.com/pingcap/docs/blob/e59c1072982e68ecd479a9619068bb658c275ef2/ticdc/ticdc-sink-to-mysql.md?plain=1#L108-L109

Reply:
Add a detailed describetion of compatibility changes in pr body and will fix official document in [this pr](https://github.com/pingcap/docs-cn/pull/12606).